notebooks/Imputation_best_practices/Imputation_best_practices.ipynb

```diff
@@ -0,0 +1,4557 @@
+{
```

Comment:
<p>I'm curious, instead of artificially injecting nans, could we directly use the existing columns that actually have missing values? Like <code>MasVnrArea</code>,<code>BsmtExposure</code>,etc?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/data-science-workflows/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='81'/>

Reply:
<p>Just wanted to follow similar process for coherence, so that it is easy to correlate. Nothing specific about not using any other column</p>